charts/zitadel/templates/deployment.yaml

```diff
@@ -189,8 +189,9 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 14 }}
-      {{- if .Values.zitadel.selfSignedCert.enabled }}
+      
       initContainers:
+        {{- if .Values.zitadel.selfSignedCert.enabled }}
```

Comment:
Was this change intentional? Before the `initContainers` stanza was left out when `Values.zitadel.selfSignedCert.enabled` was false.

Reply:
It was intentionally although there may be better solutions. Both the `generate-self-signed-cert` and the `cloudSqlProxy` require the stanza if any of them are enabled, hence the move of it. As an empty stanza to my knowledge are fine, this was the solution that I could figure out that support all scenarios of whether _either_, _both_ or _none_ of them are enabled.